packages/salesforcedx-vscode-core/test/vscode-integration/notifications/progressNotification.test.ts

```diff
@@ -24,10 +24,7 @@ describe('Progress Notification', () => {
   beforeEach(() => {
     tokenSource = new vscode.CancellationTokenSource();
     execution = new CliCommandExecutor(
-      new SfdxCommandBuilder()
-        .withArg('force')
-        .withArg('--help')
-        .build(),
+      new SfCommandBuilder().withArg('force').withArg('--help').build(),
```

Comment:
Why do we want to run `sf force --help` here?  This is what I see when I run the command in Terminal.

<img width="1257" alt="Screenshot 2024-03-21 at 5 29 10 PM" src="https://github.com/forcedotcom/salesforcedx-vscode/assets/139700604/fcc3cb9b-9f7b-422f-aae2-a4c61f3bc591">